### DISCUSSION TITLE
Can you spectate other players?

### DISCUSSION THREAD

**Discussion by roxandtol** [ppy/osu#21998] [Q&A] [answered]
I tried searching on the migration guide, and I only found multiplayer room spectating, and I wanted to know if there's a way to spectate someone playing solo. And if there's a way, does it only allow spectating players on lazer or also players on stable?

**Comment by bdach** [comment, marked as answer]
There is, but it's a bit hidden (it's in the "dashboard" overlay, "currently playing" tab).

It only allows spectating other lazer players right now.
